src/components/main/Main.scss

```
@@ -22,9 +31,19 @@
   }
 }
 
-@media screen and (min-width: $breakpoint-m) {
+@media screen and (max-width: $breakpoint-xl) {
+  .main-container {
+    margin-left: $spacing-m;
+    margin-right: $spacing-m;
+  }
+}
+
+@media screen and (max-width: 840px) {
```

I changed the last lines to this (compared to master) and it seemed to fix some of the problems: 
```suggestion
@media screen and (min-width: $breakpoint-m) {
  .main-container {
    padding: $spacing-m;
    padding-left: $spacing-m;
    padding-right: $spacing-m;
  }
}

@media screen and (min-width: $breakpoint-xl) {
  .main-container {
    max-width: $container-width-xl;
    padding-left: 0;
    padding-right: 0;
  }
}
```

There might be some breakpoint that I have missed tough 🤔

---

This seems like a more elegant solution, let's use it instead.